common-ui/src/main/res/values/styles.xml

```diff
@@ -393,7 +393,6 @@
     </style>
 
     <style name="LocationPermissionDialogTitle">
```

Comment:
Will that be a problem for existing dialogs?

Reply:
Not sure what you mean here 🤔

Reply:
If we are using a style that belongs to another Dialog, and changing some attributes. The other dialog will be affected. I assume we want this change in both.

Going to assume this is changed on a future PR. Please confirm if not.

Reply:
Actually, this specific style name hasn't been change. I'll change it in [Remove previous Location screen PR](https://github.com/duckduckgo/Android/pull/2296) 👍